nvm.sh

```diff
@@ -1954,6 +1955,51 @@ nvm() {
       fi
       return $?
     ;;
+    "upgrade" )
+      local base_version
+      base_version="$(nvm_ls_current)"
+
+      if [ $# -ge 2 ]; then
+        base_version="$(nvm_match_version $2)"
+
+        if [ $base_version = "N/A" ]; then
+          nvm_err 'Cannot find the specified version'
+          return 1
+        fi
+      fi
+
+      if [ $base_version = "none" ]; then
+        nvm_err 'No version currently active, cannot upgrade'
+        return 1
```

Comment:
I noticed that in other parts of the code, but wasn't sure if there was a convention behind it. Should exit codes be unique just within the function, or throughout the entire file?